autoload/rustfmt.vim

```diff
@@ -146,8 +146,8 @@ function! s:RunRustfmt(command, tmpname, fail_silently)
             let l:content = readfile(a:tmpname)
         endif
 
-        call writefile(l:content, expand('%'))
-        silent edit!
+        normal! ggdG
```

Comment:
Just noticed that this should become `normal! "_ggdG` to not mess with registers.